packages/client/src/utils.js

```diff
@@ -132,3 +139,14 @@ export const parseIdentifier = (identifier) => {
     contractAddress,
   };
 };
+
+export const getSafeContactName = (contacts, address) => {
+  const contact = contacts.find((contact) => contact.address === address);
+  return contact?.name;
+};
+
+export const parseTimestamp = (timestamp) => {
+  console.log("timesta", timestamp)
+  console.log("time", new Date(parseInt(timestamp) * 1000).toLocaleString())
+  return new Date(parseInt(timestamp) * 1000).toLocaleString();
```

Comment:
I think we can bring in a third party library to handle the dates and time, check out Day.js https://www.skypack.dev/view/dayjs